configs/projects/puppet-agent.rb

```diff
@@ -71,7 +70,8 @@
     end
   elsif platform.is_windows?
     # For windows, we need to ensure we are building for mingw not cygwin
-    host = "--host #{platform.platform_triple}"
+    platform_triple = platform.platform_triple
+    host = "--host #{platform_triple}"
```

Comment:
Yeah, I figured it would be better to be consistent with the other projects. This ensures we have `settings[:platform_tripple] available, and don't have to reach out to 'platform.platform_triple`